package.json

```
@@ -158,6 +158,7 @@
     "check:socks5": "mocha --config test/manual/mocharc.json test/manual/socks5.test.ts",
     "check:csfle": "mocha --config test/mocha_mongodb.json test/integration/client-side-encryption",
     "check:snappy": "mocha test/unit/assorted/snappy.test.js",
+    "check:x509-auth": "mocha test/manual/x509_auth.test.ts",
```

```suggestion
    "check:x509": "mocha test/manual/x509_auth.test.ts",
```

Since we don't use `check:ldap-auth/aws-auth/kerberos-auth`